modules/users/client/components/ProfileViewBasics.component.js

```diff
@@ -2,9 +2,30 @@ import React from 'react';
 import { NamespacesConsumer } from 'react-i18next';
 import '@/config/lib/i18n';
 import PropTypes from 'prop-types';
+import * as languages from '@/config/languages/languages';
 
-export default function ProfileViewBasics({ profile }) {
-  return (<NamespacesConsumer ns="user-profile">{ t => (<>
+export default function ProfileViewBasics({ profile, hasConnectedAdditionalSocialAccounts, isWarmshowersId, socialAccountLink }) {
+  return (<NamespacesConsumer ns={['user-profile', 'languages']}>{ (t) => (<>
+    {/* show div only if replyrate or replyTime is present */}
+    {(profile.replyRate || profile.replyTime) &&
+    <div className="profile-sidebar-section text-muted">
+      {profile.replyRate &&
+      <span>
+        {t('Reply rate {{replyRate}}.', { replyRate: profile.replyRate })}
+      </span>}
+      {profile.replyTime &&
+      <span>
+        <br/>
+        {t('Replies within {{replyTime, fromNow}}.', { replyTime: profile.replyTime })}
+      </span>}
+    </div>}
+    {/* show div only if birthdate or gender is present */}
+    {(profile.birthdate || profile.gender) &&
+    <div className="profile-sidebar-section" >
+      {profile.birthdate && t('{{birthdate, age}} years', { birthdate: new Date(profile.birthdate) })}
+      {(profile.birthdate && profile.gender) && <span>, </span>}
+      <span className={(!profile.birthdate) ? 'text-capitalize' : null }>{t(profile.gender)}.</span>
```

Comment:
For translations, we shouldn't probably pass variables as they're not picked by scripts pulling out strings for translating.

I know this structure is ported from the Angular-directive but we could improve it. Maybe something along these lines?

```jsx
const getBirthdate = (birthdate) => t('{{birthdate, age}} years', { birthdate: new Date(birthdate);

const getGender = (genderCode, sentence) => {
  switch(genderCode) {
    case 'female':
      return sentence ? t('Female') : t('female');
    // ...
  }
}

if(profile.birthdate && profile.gender) {
  sprintf( // just demonstrating, what ever equavelent of this would be in i18n-next
    t(%birthdate, %gender),
    getBirthdate(profile.birthdate),
    getGender(profile.gender, false)
  );
} else if(profile.birthdate) {
  getBirthdate(profile.birthdate);
} else if(profile.gender) {
  getGender(profile.gender, true);
}
```